renovate.json

```diff
@@ -3,7 +3,8 @@
     "schedule:weekly",
     ":rebaseStalePrs",
     ":disableMajorUpdates",
-    ":dependencyDashboard"
+    ":dependencyDashboard",
+    ":semanticCommitScope(package)"
```

Comment:
what is this? :)

Reply:
Will set the commit scope instead of using the default `deps`
https://docs.renovatebot.com/semantic-commits/#changing-the-semantic-commit-scope